### PR TITLE
Color names

### PR DESCRIPTION
**Pull request by javajudt**
Allow user to specify a color name instead of hex value--for a handful of common colors.
Display message when specified color not valid.

### CODE REVIEW

File: make-pdf.rb [hunk 1]
```diff
@@ -39,19 +54,33 @@
...
+  if COLORS[color] != nil
+    # User specified color name, so return hex
+    return COLORS[color]
+  elsif !color[/\H/] && color.length == 6
+    # User specified hex
+    return color
+  else
+    # Something's not right
+    puts "Invalid color specified."
+    exit
```
Comment (ololobus) [inline make-pdf.rb]: I guess that we never get into this code path, since `color` is either in `COLORS` or not

Comment (ololobus) [reply make-pdf.rb]: Oh, sorry, another if is more complex, but I'm still not sure that `color.length == 6` is good here. I didn't check, but probable `prawn` accepts also short hex color names, e.g. `#fff` or `#111` like in CSS, does it?

**Review by ololobus — CHANGES_REQUESTED** [review]
Another thing is that I like auto formatters, but I prefer to use single quotes in ruby for simple strings and double quotes for complex strings with substitutions and escaping. Can you configure `rufo` to stick to that behaviour?

File: make-pdf.rb (hunk 1 above)
Comment (javajudt) [reply make-pdf.rb]: Ok, I fixed the quotes. Not sure how to configure rufo to do that automatically, so I just did find and replace. 😃 

Good point, I didn't think about the short hex colors. I just tried a few of them, and it throws an error that says "unknown type of color". So it apparently does not support it.

**Review by ololobus — APPROVED** [review]

**Pull request merged**